akka-testkit/src/main/scala/akka/testkit/TestKit.scala

```diff
@@ -688,6 +724,48 @@ trait TestKitBase {
     TestKit.shutdownActorSystem(actorSystem, duration, verifySystemShutdown)
   }
 
+  // actorOf is defined without defaults for Java API compatibility
+  /**
+   * Spawns an actor as a child of this test actor, and returns the child's ActorRef.
+   * The child will stop on failure.
+   */
+  def actorOf(props: Props): ActorRef = {
```

Comment:
should perhaps be named something else that explains what it does, such as `createTestActorChild`